packages/katello/katello-repos/katello-repos.spec

```diff
@@ -68,6 +68,9 @@ rm -rf %{buildroot}
 %config %{repo_dir}/*.repo
 
 %changelog
+* Fri Jul 2 2021 James Jeffers <jjeffers@redhat.com> - 4.2-0.2.nightly
```

Comment:
You are missing a bump to release on line 8 above.

Reply:
I took the liberty to commit that, to get the builds further :)